src/functionalTest/resources/feature-toggle-with-roles.json

```diff
@@ -0,0 +1,9 @@
+{
+  "description": "Feature toggle for test",
+  "enable": true,
+  "permissions": [
```

Comment:
perhaps this file name, and headers should be updated to use 'permissions' instead of 'roles'?